examples/mysql-prerequisite/create_stored_procedure.bal

```diff
@@ -4,7 +4,8 @@ import ballerinax/mysql.driver as _;
 
 // Initializes the database as a prerequisite to `Database Access - Call stored procedures` sample.
 public function main() returns sql:Error? {
-    mysql:Client mysqlClient = check new (user = "root", password = "Test@123");
+    mysql:Client mysqlClient = check new (host = "localhost", port = 3306, user = "root",
+                                          password = "Test@123");
```

Comment:
We can do the same here also?

```ballerina
 self.db = check new ("localhost", "root", "Test@123", "MUSIC_STORE", 3306);
```

Reply:
Fixed in af73cde.

We can do in this prerequiste file. However, setup_database.bal we cannot use this, as we havent created database yet.